pkg/appconsts/versioned_consts.go

```diff
@@ -2,10 +2,11 @@ package appconsts
 
 import (
 	v1 "github.com/celestiaorg/celestia-app/pkg/appconsts/v1"
+	v2 "github.com/celestiaorg/celestia-app/pkg/appconsts/v2"
 )
 
 const (
-	LatestVersion = v1.Version
+	LatestVersion = v2.Version
 )
 
```

Comment:
[not blocking][question] do we also want to add support for version 2 below like:

```go
func SubtreeRootThreshold(version uint64) int {
	switch version {
	case v1.Version:
		return v1.SubtreeRootThreshold
	case v2.Version:
		return v2.SubtreeRootThreshold
	default:
		panic(fmt.Sprintf("unsupported version: %v", version))
	}
}

func SquareSizeUpperBound(version uint64) int {
	switch version {
	case v1.Version:
		return v1.SquareSizeUpperBound
	case v2.Version:
		return v2.SquareSizeUpperBound
	default:
		panic(fmt.Sprintf("unsupported version: %v", version))
	}
}
```
```

Reply:
noooooooooo noooo panics lol

Reply:
I had a discussion with @evan-forbes where we decided intentionally not to panic for unknown versions. 

I did think about adding the v2 values as the default but since they are currently the same, I thought to leave it as is although we should probably still make this change at some point even if to just be explicit about it